olive/platform_sdk/qualcomm/runner.py

```diff
@@ -0,0 +1,94 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import logging
+import time
+
+from olive.common.utils import run_subprocess
+from olive.platform_sdk.qualcomm.snpe.env import SNPEAndroidEnv, SNPESDKEnv
+from olive.platform_sdk.qualcomm.utils.raw_adb import run_adb_command
+
+logger = logging.getLogger(__name__)
+
+
+class SDKRunner:
+    def __init__(
+        self,
+        platform,
+        cmd: str,
+        dev: bool = False,
+        runs: int = 1,
+        sleep: int = 0,
+        log_error: bool = True,
+        android_target: str = None,
```

Comment:
Should there be an option to provide `push_to_platform` option for android? Otherwise, it always uses the default value for `True`.

Reply:
Removed the new adb changes and keep old one.